tests/serialization/test_dag_serialization.py

```diff
@@ -981,11 +980,7 @@ def test_dag_params_roundtrip(self, val, expected_val):
         assert "params" in serialized_dag["dag"]
 
         if val and any([True for k, v in val.items() if isinstance(v, set)]):
-            with pytest.warns(
-                RemovedInAirflow3Warning,
-                match="The use of non-json-serializable params is deprecated and will be removed in a future release",
-            ):
-                deserialized_dag = SerializedDAG.from_dict(serialized_dag)
+            deserialized_dag = SerializedDAG.from_dict(serialized_dag)
```

Comment:
I think the whole `if/else` should be removed. Both code path are the same.
```
    deserialized_dag = SerializedDAG.from_dict(serialized_dag)
```

Reply:
And another ups. Thanks for finding. How could I over-look this... :-(